requirements.txt

```diff
@@ -1,2 +1,3 @@
 inmanta-lsm
 pytest-inmanta==2.8.0
+devtools
```

Comment:
This dependency should also be added here: https://github.com/inmanta/pytest-inmanta-lsm/blob/bcc242c9f4b44f96d3dd278c4df12d5060d793aa/setup.py#L45